src/engines/html5/html5.js

```diff
@@ -78,6 +79,7 @@ export default class Html5 extends FakeEventTarget implements IEngine {
    * @static
    */
   static _el: HTMLVideoElement;
+  _droppedFramesWatcher: ?droppedFramesWatcher = null;
```

Comment:
1. Define it upper the static members with all others instance members.
2. Why define it as nullable if you don't do it anywhere?